.github/workflows/release.yml

```
@@ -0,0 +1,274 @@
+          fetch-depth: 0
+          ref: ${{ inputs.branch }}
+
+      - name: Configure Git
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+        shell: bash
+
+      - name: Create release branch
+        id: create-branch
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:8:31: Double quote to prevent globbing and word splitting

(shellcheck)